app/mailers/user_mailer.rb

```diff
@@ -4,6 +4,6 @@ class UserMailer < ApplicationMailer
   def welcome_email
     @user = params[:user]
```

Comment:
Just a comment for next PRs, if you are now getting the email, maybe you should rename the instance var as @username, or @user_email.